packages/rbac-lookup/PKGBUILD

```diff
@@ -0,0 +1,44 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=rbac-lookup
+pkgver=237.0cfc20b
+pkgrel=1
+pkgdesc='RBAC Lookup is a CLI that allows you to easily find Kubernetes roles and cluster roles bound to any user.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-scanner' 'blackarch-recon')
+url='https://github.com/FairwindsOps/rbac-lookup'
+license=('Apache')
+depends=()
+makedepends=('git' 'go')
+source=("git+https://github.com/FairwindsOps/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  echo "$(git rev-list --count HEAD).$(git rev-parse --short HEAD)"
```

Comment:
git tags available, please use the tags version

Reply:
Will fix.